rosidl_runtime_py/rosidl_runtime_py/convert.py

```diff
@@ -33,13 +33,16 @@ def __represent_ordereddict(dumper, data):
     return yaml.nodes.MappingNode(u'tag:yaml.org,2002:map', items)
 
 
-def message_to_yaml(msg: Any, truncate_length: int = None) -> str:
+def message_to_yaml(msg: Any, truncate_length: int = None,
+                    no_arr: bool = False, no_str: bool = False) -> str:
     """
     Convert a ROS message to a YAML string.
 
     :param msg: The ROS message to convert.
     :param truncate_length: Truncate values for all message fields to this length.
         This does not truncate the list of message fields.
+    :param no_arr: Exclude array fields of the message
```

Comment:
@vinnamkim missing period at the end of the sentence.

Reply:
Fixed.